code/modules/clothing/under/_under.dm

```diff
@@ -1,5 +1,6 @@
 /obj/item/clothing/under
 	name = "under"
+	icon = 'icons/clothing/under/jumpsuits/jumpsuit.dmi'
```

Comment:
i'd suggest making it some black-pink checkerboard jumpsuit so you can see things that don't have a REAL icon set.

Reply:
As mentioned on Discord, I will do this for `/obj/item/clothing` when the conversions are finished, that way I I can put a state for every single slot in there.